packages/api/src/coinbase.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  getUSDRate: async () => {
+    const response = await fetch('https://api.coinbase.com/v2/exchange-rates?currency=DASH', {
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json'
+      }
+    })
+
+    if (response.status !== 200) {
+      throw new Error('Coinbase api error')
+    }
+
+    return Number((await response.json()).data.rates.USD)
```

Comment:
I think you are missing null handling, in case different structure of response data returned with the status code 200. for example a lot of backend servers like to put error message in there